### DISCUSSION TITLE
Temporal：使用熟悉的编程语言实现微服务工作流编排

### DISCUSSION THREAD

**Discussion by yujinyan** [yujinyan/blog#167] [Comments]
https://blog.yujinyan.me/posts/temporal-introduction/

**Comment by Elip-kio** [comment]
> 如果考虑服务间系统健壮性，视频转码服务可能会用一个消息队列来接收视频转码的任务，文章发布服务在`落苦`文章数据后


好像是个typo🤔

**Comment by yujinyan** [comment]
> > 如果考虑服务间系统健壮性，视频转码服务可能会用一个消息队列来接收视频转码的任务，文章发布服务在`落苦`文章数据后
> 
> 好像是个typo🤔

感谢🙇‍，fixed~
